### DISCUSSION TITLE
Bug:Windows 10 style start menu app display incomplete.

### DISCUSSION THREAD

**Discussion by ShawnO28** [valinet/ExplorerPatcher#1400] [General]
The Windows 10 start menu style applied by ExplorerPatcher is not fully displayed, for example, applications such as Microsoft Store(This is an application that comes with the system) , TranslucentTB (This is an app downloaded from the Microsoft Store) ,etc. are notshowing up.

**Comment by arcyc** [comment]
All Store apps are not displayed, which looks like a known issue: https://github.com/valinet/ExplorerPatcher/issues/1108
